Sources/SQLKit/Query/SQLQueryString.swift

```diff
@@ -0,0 +1,69 @@
+public struct SQLQueryString {
+    
+    public enum Fragment {
+        case literal(String)
+        case value(Encodable)
+    }
+    
+    var fragments: [Fragment]
+}
+
+extension SQLQueryString: ExpressibleByStringLiteral {
+    public init(stringLiteral value: String) {
+        fragments = [.literal(value)]
+    }
+}
+
+extension SQLQueryString: ExpressibleByStringInterpolation {
+    public typealias StringInterpolation = SQLQueryString
```

Comment:
We can probably omit this line since it will be inferred by the `init` method, right?